ursula_cli/shell.py

```diff
@@ -195,7 +195,8 @@ def _run_heat(args, hot):
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': (os.environ.get('OS_TENANT_NAME') or
+                        os.environ['OS_PROJECT_NAME']),
```

Comment:
Which I believe will wind up setting tenant_name to None, which will have a different, more confusing error down the line 😄